app/android.pri

```diff
@@ -74,11 +74,11 @@ android {
     LIBS += -lsqlite3 -liconv -lz -lzip
     LIBS += -ltiff
     LIBS += -lwebp -lcurl -ljpeg
-    
+
     QT += multimedia
     QT += printsupport
-    QT += androidextras
     QT += bluetooth
+    QT += core-private
```

Comment:
looks scary? why it is needed?

Reply:
According to https://www.qt.io/blog/qt-extras-modules-in-qt-6 and https://doc-snapshots.qt.io/qt6-6.3/qtandroidprivate.html we need it as we use lot of the functions which are not yet available in the QtCore